Modules/Forum/classes/class.ilForumProperties.php

```diff
@@ -28,7 +28,6 @@ class ilForumProperties
     public const VIEW_DATE = 2;
     public const VIEW_DATE_ASC = 2;
     public const VIEW_DATE_DESC = 3;
-    public const FORUM_OVERVIEW_WITH_NEW_POSTS = 0;
     public const FORUM_OVERVIEW_NO_NEW_POSTS = 1;
```

Comment:
Do we still need this constant?

Reply:
removed also https://github.com/ILIAS-eLearning/ILIAS/pull/5204/files#diff-0bd3ae89f6931a13761fd65cd3b5852291a64928eff651f43e81d2c90dcfa284L48 since it was wasnt used anywhere